### PR TITLE
Climber fixes

### PR DESCRIPTION
**Pull request by EQMoore**
Adds solenoid to climber and puts updateInputs in periodic

### CODE REVIEW

**Review by ShayanHasNo — COMMENTED** [review]
LGTM too me but get someone else to double check

**Review by Aradig26 — APPROVED** [review]
LGTM

File: src/main/java/frc/robot/subsystems/climber/Climber.java [hunk 1]
```diff
@@ -21,8 +21,10 @@ public Climber(ClimberIO climberIO) {
     super();
 
     this.climberIO = climberIO;
+  }
 
-    climberIO.updateInput(climberIOInputs);
+  public void periodic() {
+    climberIO.updateInputs(climberIOInputs);
```
Comment (admlvntv) [inline src/main/java/frc/robot/subsystems/climber/Climber.java:27]: Make sure to do a processInputs too

**Review by admlvntv — CHANGES_REQUESTED** [review]
One thing

File: src/main/java/frc/robot/subsystems/climber/Climber.java (hunk 1 above)
Comment (Aradig26) [reply src/main/java/frc/robot/subsystems/climber/Climber.java:27]: Good catch

**Comment by EQMoore** [comment]
Changed to NEOs, refer to split climber

**Pull request closed without merging**